aws/resource_aws_codebuild_project.go

```diff
@@ -386,10 +388,7 @@ func resourceAwsCodeBuildProjectRead(d *schema.ResourceData, meta interface{}) e
 		return err
 	}
 
-	if err := d.Set("source", flattenAwsCodebuildProjectSource(project.Source)); err != nil {
-		return err
-	}
-
+	d.Set("source", flattenAwsCodebuildProjectSource(project.Source))
```

Comment:
I think the error check can be actually helpful here. 😺 

The convention we tend to follow is to have an error check for any fields which have nested fields under them. Not necessarily simple `[]string` but anything that has a potential of breaking - which is this case IMO.

Reply:
Makes sense! I'll get that back in there. 🚀